packages/cloud_security_posture/changelog.yml

```
@@ -5,6 +5,11 @@
 # 1.3.x - 8.8.x
 # 1.2.x - 8.7.x
 
+- version: "1.6.0-preview24"
```

if we want to include this change in 8.10
```suggestion
- version: "1.5.0-preview24"
```